pkg/tests/apis/dashboard/integration/api_validation_test.go

```diff
@@ -56,11 +57,14 @@ type TestContext struct {
 	OrgID                     int64
 }
 
-// TestIntegrationValidation tests the dashboard K8s API
-func TestIntegrationValidation(t *testing.T) {
+// TestIntegrationDashboardAPIValidation tests the dashboard K8s API with validation checks
+func TestIntegrationDashboardAPIValidation(t *testing.T) {
 	if testing.Short() {
 		t.Skip("skipping integration test")
 	}
+	if !db.IsTestDbSQLite() {
```

Comment:
@stephaniehingtgen FYI: we are disabling the execution of these integration tests against postgres and mysql. 
We are already doing something similar for Folder Integration Tests.
This is because tests are timing out, reaching more than 5 minutes to complete, since we are executing each test case on all 6 dual writer modes * 3 different databases

Please, let me know if you have any questions or concerns about this change.

Reply:
After a round of troubleshooting I identified that the reuse of the admin client to create dashboards and then cleanup them was the culprit for the slowness.

The first calls using the admin client took 5ms to complete, but after 7 or 6 calls, the performance got to 190ms.

In order to unblock our tests, I am requesting a new admin client everytime we need to create a dashboard for tests.

We still need to investigate why this is happening. I am planning to address that on a follow up PR.

For now, dashboard integration tests are finishing under 180 seconds.